MementoMori/mementomori.client/src/homePage/MainHeader.tsx

```diff
@@ -7,84 +6,124 @@ import ListItemIcon from '@mui/material/ListItemIcon';
 import Divider from '@mui/material/Divider';
 import IconButton from '@mui/material/IconButton';
 import Tooltip from '@mui/material/Tooltip';
-import PersonAdd from '@mui/icons-material/PersonAdd';
 import Settings from '@mui/icons-material/Settings';
 import Logout from '@mui/icons-material/Logout';
-import DeckMenu from './DeckMenu.tsx';
 import Button from '@mui/material/Button';
+import HomeIcon from '@mui/icons-material/Home';
+import Breadcrumb from './Breadcrumb';
+import React from 'react';
 import { AuthDialog } from '../AuthDialog/AuthDialog.tsx';
+import { useState, useEffect } from 'react';
 
-export default function MainMenu() {
+export default function MainHeader() {
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
```

Comment:
Currently this can be set to true only if user was already logged in when opening the page? Tho that doesnt seem to work either?. See if u can fix it. Also should be set to true when user logs in/registers via dialog.